src/detect_breakends.cpp

```diff
@@ -126,7 +126,13 @@ int main(int argc, char ** argv)
         return -1;
     }
 
-    seqan3::debug_stream << "Methods to be used: " << args.methods << std::endl;
+    // Check method selection uses not more than all methods.
+    if (args.methods.size() > detecting_methods::SIZE)
+    {
+        seqan3::debug_stream << "[Error] More methods were selected than exist. There may be duplications.\n";
```

Comment:
```suggestion
    // Check that method selection contains no duplicates.
    std::vector<detecting_methods> unique_methods{args.methods};
    std::ranges::sort(unique_methods);
    unique_methods.erase(std::unique(unique_methods.begin(), unique_methods.end()), unique_methods.end());
    if (args.methods.size() > unique_methods.size())
    {
        seqan3::debug_stream << "[Error] The same detection method was selected multiple times.\n";
```

Otherwise, you can have duplications as long as you do not select more than 4.

Reply:
thanks you are of course right, that was lazy programming 😇